charts/proxy/templates/deployment.yaml

```diff
@@ -32,8 +32,9 @@ spec:
           value: {{ .Values.storage.mongo.url | quote }}
         {{- end }}
         # TODO: re-enable when workers are used
+        # SPACES of template must be fixed back while re-enabling
         #- name: ATHENS_REDIS_QUEUE_ADDRESS
-        #  value: {{ .Values.redis.address | quote }}
+        #  value: { { .Values.redis.address | quote } }
```

Comment:
@fedepaol what was wrong with this before?

Reply:
@arschles see #715 . Basically helm was trying to render the template even if it was inside a comment